deploy/openshift/templates/README.md

```diff
@@ -0,0 +1,155 @@
+## Context
+
+All commands are executed in the directory where this README is located.
+
+## Templates and Parameters
+
+Templates are provided with a set of predefined params which you can override with with `-p key=value`. If you miss envs and parameters, you can add them to your template.
+
+IMPORTANT! **ROUTING_SUFFIX** value in the below commands `$(minishift ip).nip.io` works for MiniShift only. You MUST provide **own routing suffix** when deploying to OCP cluster.
```

Comment:
It would be useful to have a command to get this value if there is a command for that.

Reply:
I wish I knew such a command :)

Reply:
could we create a dummy service then we got the default routing suffix and then we can display/reuse it ?

Reply:
Not sure I understand. Dummy service for what?

Reply:
@eivantsov If I create a dummy application on openshift like nginx app and I create a service nginx and expose port 80, openshift will provide me an URL. From this URL I can guess the "routing suffix" so user has a way to find it (through a command that can try to create app, create service and route and then display the url and delete project)

It's just to have a way to provide to users "hey we can give you some hints around this routing suffix"

Reply:
A lame workaround may be updating Che dc after routes for Che and Keycloak are created. This will add complexity to commands, but this way, no input if required from a user

Reply:
in my approach, it was a way to see what the route could be.
in your approach, we create using defaults, grab values and update again.

in both case it would help "newcomers" so I'm fine.

Reply:
Added a sub-section on How to get routing suffix.